s3/src/main/scala/org/apache/pekko/stream/connectors/s3/model.scala

```diff
@@ -194,6 +200,12 @@ final class BucketVersioning private (val status: Option[BucketVersioningStatus]
   /** Java API */
   def getMfaDelete: java.util.Optional[MFAStatus] = mfaDelete.asJava
 
+  /** Java API */
+  def getBucketVersioningEnabled: Boolean = bucketVersioningEnabled
```

Comment:
With Java APIs, it is more common to use 'is' instead of 'get' for the names of getters that return booleans.